coreos-with-fleet-multivm/metadata.json

```diff
@@ -0,0 +1,7 @@
+{ 
+  "itemDisplayName": "Deploy a CoreOS cluster hosting Fleet", 
```

Comment:
Suggestion for name so that it shows up better in the Azure.com index: CoreOS cluster hosting fleet